Cargo.toml

```diff
@@ -34,6 +34,10 @@ egui_tabs = { git = "https://github.com/damus-io/egui-tabs", rev = "6eb91740577b
 egui_virtual_list = { git = "https://github.com/jb55/hello_egui", rev = "a66b6794f5e707a2f4109633770e02b02fb722e1" }
 ehttp = "0.5.0"
 enostr = { path = "crates/enostr" } 
+egui-video = { path = "../egui-video-rs" }
```

Comment:
this is referencing some unknown crate that is outside the project. will need to be a regular crate reference on github like all of our other deps